### ISSUE TITLE
batchnormalization issue

### ISSUE THREAD

**Issue by xuchengggg** [keras-team/keras#13232]
I use `class BatchNorm(keras.layers.BatchNormalization):`
`    def call(self, inputs, training=None):`
`        return super(self.__class__,self).call(inputs, training=training)`
as bn layer.
On training time, batch_size is 8, and I set the training=True, I am not sure if training should be set to true or false when testing which can get totally different results.
